### ISSUE TITLE
Allow to pass filepath:line:column for descriptionMoreURL

### ISSUE THREAD

**Issue by sadovnychyi** [atom-community/autocomplete-plus#424]
Allow to pass filepath:line:column for `descriptionMoreURL` to be able to jump to definition with possible doc strings.

**Comment by joefitzgerald** [comment]
This is a great idea. :+1:

**Comment by jlegewie** [comment]
Yes, great idea. In general, I would prefer a more flexible system. Currently, only URLs are allowed. Why not allow a callback function that could do anything? For example, 'More' could open a popup with the function definition without relying on a browser. 'More' could handle anything not just URLs or `file path:line:column`.

Second (I can open a separate issue for that), it would be nice if the user can open `More` with a key-combo and not just by clicking on `More..`

**Comment by benogle** [comment]
> Why not allow a callback function that could do anything?

I'd like to keep the experience consistent between providers. We should come up with the best way to display/interact with the docs, and make it easy for all providers to get it.

I think adding the ability for a path:line:col is great.

> it would be nice if the user can open More with a key-combo and not just by clicking on More..

This is a great idea too

**Comment by jlegewie** [comment]
The consistent experience sounds like the right plan. Personally, I think it would be good if the docs are shown in Atom itself and not by an external application.

**Comment by sandeepmahanty** [comment]
Hi, Is this open? I wish to work on this. Would like to know where to begin with to make changes.

**Comment by tim-sorrels** [comment]
Hello, is this still open?

**Comment by icecream17** [comment]
Yes it's still open
Note that you're actually looking for https://github.com/atom/snippets
